src/oidcendpoint/oidc/userinfo.py

```diff
@@ -34,15 +36,16 @@ class UserInfo(Endpoint):
 
     def __init__(self, endpoint_context, **kwargs):
         Endpoint.__init__(self, endpoint_context, **kwargs)
```

Comment:
Shouldn't `add_claims_by_scope` default to True for userinfo?
Something like this?
```python
    def __init__(self, endpoint_context, add_claims_by_scope=True, **kwargs):
        Endpoint.__init__(
            self,
            endpoint_context,
            add_claims_by_scope=add_claims_by_scope,
            **kwargs,
        )
```

Reply:
Probably